### ISSUE TITLE
Dev Test: Molecules and Light 1.5.0-dev.50

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#503]
@ariel-phet, @katiewoe molecules-and-light/1.5.0-dev.50 is ready for dev testing. We are preparing this sim for publication with the suite of a11y features including sound, alternative input, screen reader access, and zoom. New simulation features also include new molecule "Methane" and timing controls for "slow motion" play. Please test all of these features in particular.

It would be helpful to see how these features are behaving on all of PhET's supported platforms before we move to RC. Please make sure that the sim is behaving correctly on as many platforms as possible.

Assigning @ariel-phet for prioritization.

Thanks!

<details>
<summary><b>General Dev Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Try to include browser version numbers
- If there is a console available, check for errors and include them in the Problem Description.
- Wrung through the string tests on at least one platform, especially if it is about to go to rc.

<h3>General Dev Test Platforms</h3>

- [x] Latest macOS, All browsers
- [x] Latest iOS, Safari
- [x] Windows 10, Chrome
- [x] Windows 10, Edge
- [x] Windows 10, Firefox
- [x] Latest Chrome OS, Chrome
~~- [ ] Windows 10, IE11~~ (EDIT: This is about to be dropped as a supported platform)

These issues should have either use the labels "status:ready-for-qa" or "status:ready-for-review." If it is ready for QA then close the issue if fixed. If ready for review then leave open and assign back to the developer.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.50/phet/molecules-and-light_all_phet.html?zoom=true)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) Dev Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature testing doesn't negatively affect the sim in any way.
- Load the a11y view and make sure that interacting with all elements in the simulation updates the appropriate descriptions in the PDOM.

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.50/phet/molecules-and-light_a11y_view.html?zoom=true)**
- **[Simulation](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.50/phet/molecules-and-light_all_phet.html?zoom=true)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [ ] **macOS + Safari + VoiceOver**
- [x] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in [this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a screen reader.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>

  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary>

  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>

  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>

  We typically assign the developer who opened the issue in the QA repository.

  </details>

  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>

  You should:

  1. Consult the [QA Book](link).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by ariel-phet** [comment]
@KatieWoe high priority, but after #501

**Comment by KatieWoe** [comment]
Repo for issues: https://github.com/phetsims/molecules-and-light/issues

**Comment by KatieWoe** [comment]
zoom methods:
1 - keyboard navigation with ctrl +, -
2 - Pinch to zoom with fingers
3 - ctrl + mouse wheel

**Comment by KatieWoe** [comment]
QA is done

**Comment by terracoda** [comment]
@KatieWoe, I hear "unpronouncable", too, on mobile and I hear "white bullet" on MacBook Pro. 

Unfortunately, removing the visual bullets with CSS removes the list semantics all together in Safari and then learners using VO lose list semantics and the list count. This is a Safari issue (https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)

@jessegreenberg, we can discuss testing the work around in the article at MAL's design meeting.

**Comment by jessegreenberg** [comment]
Thanks QA team, this has been done for a while, closing.
